mindsdb/integrations/handlers/tripadvisor_handler/tripadvisor_api.py

```diff
@@ -0,0 +1,165 @@
+            radius (int): Length of the radius from the provided latitude/longitude pair to filter results.
+            radiusUnit (str): Unit for length of the radius. Valid options are "km", "mi", "m" (km=kilometers, mi=miles, m=meters.
+            language (str): The language in which to return results (e.g. "en" for English or "es" for Spanish) from the list of our Supported Languages.
+
+        Returns:
+            response: Response object with response data as application/json
+        """
+
+        url = url + "search?language={language}&key={api_key}&".format(
+            api_key=self.api_key, language=language
+        )
+
+        url = self.processQuery(url, params_dict)
+        response = self.getResponse(url)
+
+        return response
+
+    def location_details(
+        self, url: str, params_dict: dict, locationId: str, language: str = "en"
+    ) -> Response:
+        """
+        A Location Details request returns comprehensive information about a location (hotel, restaurant, or an attraction) such as name, address, rating, and URLs for the listing
+        on Tripadvisor.
+
+        Args:
+            locationId (str): A unique identifier for a location on Tripadvisor. The location ID can be obtained using the Location Search.
+            language (str): The language in which to return results (e.g. "en" for English or "es" for Spanish) from the list of our Supported Languages.
+            currency (str): The currency code to use for request and response (should follow ISO 4217).
+
+        Returns:
+            response (Response): Response object with response data as application/json
+        """
+        url = url + "{locationId}/details?language={language}&key={api_key}&".format(
+            locationId=locationId, api_key=self.api_key, language=language
+        )
+        url = self.processQuery(url, params_dict)
+        response = self.getResponse(url)
+
+        return response
+
+    def location_reviews(
+        self, url: str, locationId: str, language: str = "en"
+    ) -> Response:
+        """
+        The Location Reviews request returns up to 5 of the most recent reviews for a specific location. Please note that the limits are different for the beta subscribers.
+
+        Args:
+            locationId (str): A unique identifier for a location on Tripadvisor. The location ID can be obtained using the Location Search.
+            language (str): The language in which to return results (e.g. "en" for English or "es" for Spanish) from the list of our Supported Languages.
+
+        Returns:
+            response: Response object with response data as application/json
+        """
+        url = url + "{locationId}/reviews?language={language}&key={api_key}&".format(
+            locationId=locationId, api_key=self.api_key, language=language
+        )
+
+        response = self.getResponse(url)
+        return response
+
+    def makeRequest(self, apiCall, **params):
```

Comment:
Can you please change the name to be consistent to other methods?

Reply:
Alright will do!